package.json

```diff
@@ -49,6 +49,7 @@
     "babel-loader": "^8.0.6",
     "core-js": "^3.2.1",
     "coveralls": "^3.0.3",
+    "delatin": "^0.2.0",
```

Comment:
You shouldn't have `delatin` in the top-level `package.json`; just the module-level `package.json`

Reply:
got it